geostore/check_files_checksums/utils.py

```diff
@@ -19,14 +20,7 @@
 ARRAY_INDEX_VARIABLE_NAME = "AWS_BATCH_JOB_ARRAY_INDEX"
 
 
-class ChecksumMismatchError(Exception):
```

Comment:
It was good to get rid of this. Raising and catching an exception within a single piece of code was definitely a code smell.